python/singa/net.py

```diff
@@ -53,13 +53,20 @@
     y = net.predict(x)
     print tensor.to_numpy(y)
 """
-
+from __future__ import print_function
+from __future__ import absolute_import
+
+from future import standard_library
+standard_library.install_aliases()
+from builtins import zip
+from builtins import str
+from builtins import object
 from .proto.model_pb2 import kTrain, kEval
-from __init__ import __version__
-import tensor
-import layer
-import snapshot
-import cPickle as pickle
+from .__init__ import __version__
+from . import tensor
+from . import layer
+from . import snapshot
+import pickle as pickle
```

Comment:
since future does not handle the pickle package properly, i am going to use the idiom 

```
try:
   import pickle
except ImportError:
   import cPickle as pickle
```